### DISCUSSION TITLE
What novel begins with this phrase?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#751] [Galaxy Trivia] [answered]
*"Stately, plump Buck Mulligan came from the stairhead, bearing a bowl of lather on which a mirror and a razor lay crossed"*

**Comment by ParanoidUser** [comment]
@mo-cherif, do you know this one? 😉

**Comment by SANKARAMDAS** [comment]
John "Buck" Mulligan is a fictional character in James Joyce's 1922 novel Ulysses. 
He appears most prominently in episode 1 (Telemachus), and is the subject of the novel's famous first sentence: "Stately, plump Buck Mulligan came from the stairhead, bearing a bowl of lather on which a mirror and a razor lay crossed."

**Comment by peterbaccaro** [comment, marked as answer]
The novel is **"Ulysses" by James Joyce**. This is the opening sentence of the book and it describes the character Buck Mulligan, who is one of the main characters in the story. The novel is considered a modernist masterpiece and is known for its complex narrative style and use of stream-of-consciousness technique. It follows the story of Leopold Bloom and Stephen Dedalus as they wander through Dublin on June 16, 1904.
